src/lib/constants/networks.ts

```diff
@@ -15,9 +16,14 @@ export const NETWORK_CONFIGS: Record<Network, NetworkConfig> = {
 	[Network.MAINNET]: {
 		rpcUrl: 'https://mainnet.infura.io/v3/77c3d733140f4c12a77699e24cb30c27',
 		ipfsGateway: DEFAULT_IPFS_GATEWAY
+	},
+	[Network.GOERLI]: {
+		rpcUrl: 'https://goerli.infura.io/v3/77c3d733140f4c12a77699e24cb30c27',
+		ipfsGateway: DEFAULT_IPFS_GATEWAY
 	}
 };
 
 export const NETWORK_ETHERSCAN: Record<Network, string> = {
-	[Network.MAINNET]: ''
+	[Network.MAINNET]: '',
+	[Network.GOERLI]: ''
```

Comment:
`Network.GOERLI]: 'goerli'` - this is basically the thing to append to the start of etherscan URLs, e.g. `https://goerli.etherscan.io/`